document-generator-core/swagger-2.0/spec/document-generator.json

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document generator API",
+    "description": "An API for generation of documents",
+    "version": "1.0.0"
+  },
+  "host": "localhost",
+  "schemes": [
+    "http"
+  ],
+  "tags": [
+    {
+    "name": "document-generator"
+    }
+  ],
+  "paths": {
+    "/document-generate": {
```

Comment:
should be `private` and the URI space should be built from the resource type of `documents` followed by the action to `generate` so should be:

`/private/documents/generate`